core/libraries/messages/messenger/EE_Html_messenger.class.php

```diff
@@ -473,7 +475,8 @@ protected function _send_message()
         );
         $this->_deregister_wp_hooks();
         add_action('wp_enqueue_scripts', array($this, 'enqueue_scripts_styles'));
-        echo $this->_get_main_template(); // already escaped
+
+        echo $this->_get_main_template();
```

Comment:
can this one not be escaped?

Reply:
Since it is a full HTML page with doctype, html, meta, script, style, and other tags, we cannot escape it without destroying the required contents including but not limited to embedded JS and CSS codes that are included by any plugin using WP head and footer actions of WordPress.